tests/test_e2e.py

```diff
@@ -92,11 +97,11 @@ async def test_builtin_tool_brave_search(self):
         logs = [log async for log in run_client(client, dialog)]
         expected_logs = [
             LogEvent(StepType.shield_call, "No Violation"),
-            LogEvent(StepType.inference, "brave_search.call"),
-            LogEvent(StepType.tool_execution, "Tool:BuiltinTool.brave_search Args:"),
+            LogEvent(StepType.inference, "<function=brave_search>"),
```

Comment:
so yeah this caught me by surprise

Reply:
yeah, with `function_tag` sometimes the model ends up calling builtin tools also with `<function>` format.